src/components/include/utils/callable.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SRC_COMPONENTS_INCLUDE_UTILS_CALLABLE_H
+#define SRC_COMPONENTS_INCLUDE_UTILS_CALLABLE_H
+namespace utils {
+class Callable {
+ public:
+  virtual void operator()() const = 0;
+  virtual ~Callable() {}
+};
+
+class DummyCallable : public Callable {
```

Comment:
@LuxoftAKutsan 
I absolutely don't like this name, suggest to change to CallNothing